backend/dtos/dtos.go

```diff
@@ -51,10 +51,7 @@ type Operation struct {
 }
 
 type Query struct {
-	ID    int64  `json:"id"`
-	Name  string `json:"name"`
-	Query string `json:"query"`
-	Type  string `json:"type"`
+	ID int64 `json:"id"`
```

Comment:
I think this change is unrelated to data source change. This is simply returning more data, which it looks like the front end uses

Reply:
Yes, that's right. If you look at list_queries_for_operation, it defines this struct:
```go
type QueriesForOperationOutput struct {
	ID    int64  `db:"id" json:"id"`
	Name  string `db:"name" json:"name"`
	Query string `db:"query" json:"query"`
	Type  string `db:"type" json:"type"`
}
```
This is effectively the same data dtos.Query is providing. I think this change was made because we wanted all return types to live in the dtos package, rather than creating one off instances inside the individual services.